src/current/_data/redirects.yml

```
@@ -0,0 +1,841 @@
+# 
+# Add a new block like this:
+# - destination: new-file-name-with-html-extension.html#optional-anchor
+#   sources: ['old-file-name-with-html-extension.html', 'other-old-file-name.html']
+#   temporary: true|false (default: false)
+#   versions: ['v22.1', 'v22.2']
+#
+# Or like this:
+# - destination: new-file-name-with-html-extension.html#optional-anchor
+#   sources: 
+#     - old-file-name-with-html-extension.html
+#     - other-old-file-name.html
+#   temporary: true|false (default: false)
+#   versions: ['v22.1', 'v22.2']
+# 
+# source can support an array of pages (e.g., you have several pages
+# you wish to redirect all at once to another page)
+# status_code can be any valid status code, including 301!
+# versions is an array of folders where the page exists
+# versions, source, and destination can all take advantage of
+# placeholders within Netlify:
+#
+# https://docs.netlify.com/routing/redirects/redirect-options/#placeholders
+############################################################
+
+# /docs/v2.2/* /docs/v19.1/:splat 301!
+# /docs/managed/* /docs/cockroachcloud/:splat 301!
+# /docs/releases/ /docs/releases/index.html 301!
+# /docs/advisories/ /docs/advisories/index.html 301!
+# /docs/advisories/advisories.html /docs/advisories/index.html 301
+
+- destination: v19.1/:splat
+  sources: ['v2.2/*']
+  force: true
+
+- destination: cockroachcloud/:splat
+  sources: ['managed/*']
+  force: true
+
+- destination: releases/index.html
+  sources: ['releases/']
+  force: true
+
+- destination: advisories/index.html
+  sources:
+    - advisories/
+    - advisories/advisories.html
+  force: true
+
+# Pages undergoing maintenance
+
+- destination: migration-overview.html
+  sources: 
+    - migrate-from-mysql.html
+    - migrate-from-oracle.html
+    - migrate-from-postgres.html
+  temporary: true
+  versions: ['v21.2', 'v22.1', 'v22.2']
+
+# Renamed pages
+
+- destination: common-errors.html
+  sources: ['general-troubleshooting.html']
+  versions: [':version']
+
+- destination: cost-based-optimizer.html
+  sources: ['sql-optimizer.html']
+  versions: [':version']
+
+- destination: set-vars.html
+  sources: ['set-application-name.html', 'set-database.html', 'set-time-zone.html']
+  versions: [':version']
+
+- destination: show-vars.html
+  sources: ['show-all.html', 'show-time-zone.html', 'show-transaction.html']
+  versions: [':version']
+
+- destination: cloud-org-sso.html
+  sources: ['cloud-sso.html']
+  versions: ['cockroachcloud']
+
+- destination: connect-to-a-serverless-cluster.html
+  sources: ['connect-to-a-free-cluster.html']
+  versions: ['cockroachcloud']
+
+- destination: create-a-serverless-cluster.html
+  sources: ['create-a-free-cluster.html']
+  versions: ['cockroachcloud']
+
+- destination: learn-about-pricing.html
+  sources: ['planning-your-serverless-cluster.html']
+  versions: ['cockroachcloud']
+
+- destination: managing-access.html
+  sources: ['console-access-management.html', 'user-authorization.html']
```

```suggestion
  sources: [
      'console-access-management.html',
      'user-authorization.html'
    ]
```
WDYT for readability and maintenance?